src/+otp/Problem.m

```diff
@@ -53,7 +53,7 @@
         
         function set.Settings(obj, value)
             expNumVars = value.ExpectedNumVars;
-            if ~(isempty(expNumVars) || length(value.Y0) == expNumVars)
+            if expNumVars > 0 && length(value.Y0) ~= expNumVars
```

Comment:
I would still like expNumVars to be possibly empty, that is very standard in matlab IMO

Reply:
Not a strong opinion on it though.